### PR TITLE
add Deno support

### PR DESCRIPTION
**Pull request by ryanccn**
This PR adds Deno support for `sade` in the `deno/` subdirectory.

### CODE REVIEW

File: deno/mod.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    return this;
+  }
+
+  version(str) {
+    this.ver = str;
+    return this;
+  }
+
+  parse(arr, opts = {}) {
+    arr = ["I can't believe", "I have to do this", ...arr]; // copy with 2 additional items before for compatibility...
```
Comment (ryanccn) [inline deno/mod.js:124]: I'm not sure this is a good way to replace `process.argv` though 😅

Comment (lukeed) [inline deno/mod.js:124]:
```suggestion
    arr = ["", ""].concat(arr);
```

**Review by lukeed — APPROVED** [review]
Thanks! Will be converting to TS and readjusting some of the other changes that were made before first publish.

I got in touch w/ Deno team and the webhook has been reassigned to this repo. Thanks for the contribution :)

**Comment by ryanccn** [comment]
Cheers 🍻

**Pull request merged**